app/jobs/post_grouper.rb

```diff
@@ -0,0 +1,19 @@
+#class PostGrouper < ActiveJob::Base
+class PostGrouper
+  def perform(cleanup_gene_claims = true)
```

Comment:
For clarity I can rename `delete_orphaned_gene_claims` to `delete_orphaned_ensembl_gene_claims` and `should_cleanup_gene_claims?` to `should_cleanup_ensembl_gene_claims?`.